src/tests/actions/articleActions.test.js

```diff
@@ -178,6 +182,63 @@ describe('articleActions', () => {
     );
   });
 
+  it('should update a comment', async () => {
+    const comment = 1;
+    const slug = 'react-redux';
+    const response_data = {
+      comment: {
+        body: 'react-redux',
+      },
+    };
+    mock.onPut('/api/articles/react-redux/comments/1/').reply(200, response_data);
+    editComment(response_data, slug, comment)(store.dispatch);
+    await flushAllPromises();
+    expect(store.getActions()).toEqual(
+      [
+        { type: UPDATE_COMMENT_INITIATED, payload: true },
+        { type: UPDATE_COMMENT_SUCCESS, payload: true },
+      ],
+    );
+  });
+
+  it('should not update a comment when a user is logged in', async () => {
```

Comment:
@MuhanguziDavid Please cross-check this comment. I think it should be `when a user is not logged in`

Reply:
Thank you @samuelbwambale , naming has been changed accordingly.